### PR TITLE
Support group (bouquet) channel order from the backend

### PR DESCRIPTION
**Pull request by phunkyfish**
v4.8.0
- Added: Support group (bouquet) channel order from the backend
- Added: Extract new/live/premiere as part of show info from EPG
- Fixed: Fix possible runtime error assigning nullptr to string
- Fixed: Preserve recordings and timer lists if connection lost
- Update: Remove p8-platform dependency on Threads, Mutex, Condition, Util and OS
- Fixed: Fix long kodi shutdown delay if shutodwn happens before addon fully starts
- Update: Update regex to static/const (perf benefit)
- Update: Update show info regex's

### CODE REVIEW

**Comment by phunkyfish** [comment]
A nice small one this time ;)

**Comment by phunkyfish** [comment]
@ksooo any chance of a review if you have the time?

**Comment by phunkyfish** [comment]
I've introduced a bug in this PR, will let you know once fixed.

File: src/enigma2/Admin.h [hunk 1]
```diff
@@ -32,6 +31,8 @@
 #include <string>
 #include <vector>
 
+#include <kodi/libXBMC_pvr.h>
```
Comment (ksooo) [inline src/enigma2/Admin.h:34]: Other addons do not conisder "kodi" as system headers. Why the change?

Comment (ksooo) [reply src/enigma2/Admin.h:34]: As discussed in another PR, this change seems to be fine.

**Review by ksooo — APPROVED** [review]
Feel free to merge (once Jenkins is happy?)

**Comment by phunkyfish** [comment]
jenkins build this please
